templates/resources/elasticsearch_security_system_user.md.tmpl

```diff
@@ -0,0 +1,18 @@
+---
+subcategory: "Security"
+layout: ""
+page_title: "Elasticstack: elasticstack_elasticsearch_security_system_user Resource"
+description: |-
+  Updates system user's password and enablement.
+---
+
+# Resource: elasticstack_elasticsearch_security_system_user
+
+Updates system user's password and enablement. See, https://www.elastic.co/guide/en/elasticsearch/reference/current/built-in-users.html
+Since this resource is to manage undeletable built-in user, destroy will removes it only from terraform state and the user itself won't be deleted.
```

Comment:
```suggestion
Since this resource is to manage built-in users, destroy will not delete the underlying Elasticsearch and will only remove it from Terraform state.
```